src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -0,0 +1,645 @@
+  </data>
+  <data name="Globals_AlwaysShowTabs.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, tabs are always displayed. When unchecked and 'show the title bar' is checked, tabs only appear after opening a new tab.</value>
+  </data>
+  <data name="Globals_CompactTabWidthMode.Content" xml:space="preserve">
+    <value>Compact</value>
+  </data>
+  <data name="Globals_ConfirmCloseAllTabs.Content" xml:space="preserve">
+    <value>Show close all tabs popup</value>
+  </data>
+  <data name="Globals_ConfirmCloseAllTabs.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, closing a window with multiple tabs open will require confirmation. When unchecked, the confirmation dialog will not appear.</value>
+  </data>
+  <data name="Globals_CopyFormatting.Content" xml:space="preserve">
+    <value>Copy formatting</value>
+  </data>
+  <data name="Globals_CopyFormatting.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, the color and font formatting of selected text is also copied to your clipboard. When unchecked, only plain text is copied to your clipboard.</value>
+  </data>
+  <data name="Globals_CopyOnSelect.Content" xml:space="preserve">
+    <value>Copy after selection is made</value>
+  </data>
+  <data name="Globals_CopyOnSelect.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, a selection is immediately copied to your clipboard upon creation. When unchecked, the selection persists and awaits further action.</value>
+  </data>
+  <data name="Globals_DarkTheme.Content" xml:space="preserve">
+    <value>Dark</value>
+  </data>
+  <data name="Globals_DefaultLaunchSize.Content" xml:space="preserve">
+    <value>Default</value>
+  </data>
+  <data name="Globals_DefaultProfile.Header" xml:space="preserve">
+    <value>Default profile</value>
+  </data>
+  <data name="Globals_DefaultProfile.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Profile that opens when clicking the '+' icon or by typing the key binding assigned to 'newTab'.</value>
+  </data>
+  <data name="Globals_DefaultTheme.Content" xml:space="preserve">
+    <value>Default</value>
+  </data>
+  <data name="Globals_DisableDynamicProfiles.Content" xml:space="preserve">
+    <value>Automatically create new profiles when new shells are installed</value>
+  </data>
+  <data name="Globals_DisableDynamicProfiles.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Enables all of the dynamic profile generators, adding their profiles to the list of profiles on startup.</value>
+  </data>
+  <data name="Globals_EqualTabWidthMode.Content" xml:space="preserve">
+    <value>Equal</value>
+  </data>
+  <data name="Globals_ForceFullRepaint.Content" xml:space="preserve">
+    <value>Screen redrawing</value>

```

Comment:
This feels like it only takes effect when in full screen mode?

Reply:
Good lord.

Can we call it "Disable differential drawing" or "use differential drawing" or something and not avoid negatives? The tooltip can explain what it is.

I may unsubscribe if we're already getting into individual word-by-word bikeshedding.

Reply:
If now's not the time for nitpicking the text, when is? When we're reviewing a 200 file mega merge of all the settings UI code? I'd rather _not_, and be able to mentally check "yea, we already reviewed the resources".

I think that "Disable differential drawing" is the most accurate to what this does.

Reply:
I think we should avoid using negatives for checkboxes. I can change it to "Enable differential drawing" and flip the logic of the tooltip.

Reply:
But this is a _troubleshooting option_. It's only to be used when there's trouble to be shot. If the user opens this up and sees one of the _troubleshooting options_ turned on, what are they going to think?

As a user, I would think "that's guarded with an electric fence, I should not turn it off" or "that's weird. I'll see how it works without this _option that is to be used in case of trouble_ turned on."

I have not seen a clear articulation on why exactly we want to avoid using negatives?

Reply:
I have some guidelines [here](https://docs.microsoft.com/en-us/windows/win32/uxguide/text-ui#control-labels) that say we should use positive messaging:
```
In general, prefer positive phrasing. For example, use do instead of do not, and notify instead of do not notify.

Exception: The check box label, "Don't show this message again," is widely used.
```
On the flip side, we could rename it to something close to its actual name like "Force full repaint" or "Redraw entire screen when display updates".

Reply:
Does "disable" count as negative phrasing though?

Reply:
I'm partial to "Redraw entire screen when display updates" with a subtitle indicating that it may be helpful when troubleshooting an issue or that it will reduce performance. Every option on this page, I think, either needs that subtitle or needs the whole page to have a disclaimer about it.

We can't be surfacing these options as just normal every-day run-of-the-mill things users should be toggling. This is a product made for power users, we should treat them as though they can make decisions.

Reply:
I would argue having a negative effect on a checkbox may be confusing. I'm happy to change it to "Redraw entire screen when display updates" and add a disclaimer to the top of the page.